go.mod

```diff
@@ -1,17 +1,13 @@
 module github.com/kairos-io/immucore
 
-go 1.23.1
-
-replace (
-	github.com/onsi/ginkgo/v2 v2.17.1 => github.com/onsi/ginkgo/v2 v2.12.1
-	github.com/onsi/gomega v1.33.0 => github.com/onsi/gomega v1.30.0
-)
+go 1.24
 
 require (
 	github.com/avast/retry-go v3.0.0+incompatible
+	github.com/containerd/containerd v1.7.25
```

Comment:
this was just moved from below into the main require as we had 2 require stanzas for some reason